publish_template.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#
+# Push a new Inferno version to inferno-template GitHub Repository via SSH.
+#
+# USAGE:
+#   ./publish_template
+
+set -e
+set -x
+
+
+REPO_URL=${1:-git@github.com:inferno-framework/inferno-template.git}
+VERSION=$(bundle exec ./bin/inferno version)
+
+
+if [ -d ./inferno-template ]; then
+  echo "ERROR: detected inferno-template directory, please remove or rename it to run this script."
+  exit 1;
+fi;
+
+git clone $REPO_URL ./inferno-template
+rm -f ./inferno-template/Gemfile.lock
+
+bundle exec ./bin/inferno new inferno-template --author "Inferno Template" --force
+
+git -C ./inferno-template add --all
+git -C ./inferno-template commit --message "update inferno-template to v$VERSION"
+git -C ./inferno-template push origin main
+
+rm -rf ./inferno-template
```

Comment:
should we have a ./tmp directory that is committed and just in the repo for this type of thing (with a .keep file but also in then in the .gitignore so that nothing in there gets committed... see the data directory)?  That way this would be under /tmp/inferno-template and be very clear this is not something ever intended to live in this repo but rather just an ephemeral directory to stage changes getting pushed to git.